apps/native-component-list/src/screens/Video/VideoScreen.tsx

```diff
@@ -9,8 +9,13 @@ import { PixelRatio, ScrollView, StyleSheet, Text, View } from 'react-native';
 import Button from '../../components/Button';
 import TitledSwitch from '../../components/TitledSwitch';
 
-const bigBuckBunnySource: VideoSource =
-  'https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4';
+const bigBuckBunnySource: VideoSource = {
+  uri: 'https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4',
+  notificationData: {
```

Comment:
I didn't wan't to call it `metadata` as we aren't actually changing any metadata on the native side for iOS, but I think it will be the easiest to understand for the users this way